arangod/Network/NetworkFeature.cpp

```diff
@@ -114,6 +132,12 @@ void NetworkFeature::collectOptions(std::shared_ptr<options::ProgramOptions> opt
   options->addOption("--network.protocol", "network protocol to use for cluster-internal communication",
                      new DiscreteValuesParameter<StringParameter>(&_protocol, protos))
                      .setIntroducedIn(30700);
+
+  options
+      ->addOption("--network.max-requests-in-flight", std::string("controls the number of internal requests that can be in flight at a given point in time"),
+                  new options::UInt64Parameter(&_maxInFlight),
+                  options::makeDefaultFlags(options::Flags::Dynamic, options::Flags::Hidden))
```

Comment:
I think the `Dynamic` option is not accurate here, as the default value for the option is hard-coded and not determined dynamically via some properties of the host system.